<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSourceDlg</class>
 <widget class="QDialog" name="DataSourceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataSource</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>&amp;Type:</string>
         </property>
         <property name="buddy">
          <cstring>typeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeComboBox">
         <item>
          <property name="text">
           <string>CLIENT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TANGO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PYEVAL</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="applyPushButton">
         <property name="text">
          <string>&amp;Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetPushButton">
         <property name="text">
          <string>&amp;Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="nameFrame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>&amp;Name:  </string>
            </property>
            <property name="buddy">
             <cstring>nameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QFrame" name="dsFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="docFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="docLabel">
             <property name="text">
              <string>&amp;Doc:</string>
             </property>
             <property name="buddy">
              <cstring>docTextEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="docTextEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="closeSaveFrame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="savePushButton">
            <property name="text">
             <string>Sa&amp;ve to File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="storePushButton">
            <property name="text">
             <string>St&amp;ore in DB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closePushButton">
            <property name="text">
             <string>&amp;Close</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>typeComboBox</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>docTextEdit</tabstop>
  <tabstop>savePushButton</tabstop>
  <tabstop>storePushButton</tabstop>
  <tabstop>closePushButton</tabstop>
  <tabstop>applyPushButton</tabstop>
  <tabstop>resetPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
